MapboxGeocoder/MBPlacemark.swift

```diff
@@ -108,6 +114,12 @@ open class Placemark: NSObject, Codable {
             let northEast = CLLocationCoordinate2D(geoJSON: Array(boundingBox.suffix(2)))
             region = RectangularRegion(southWest: southWest, northEast: northEast)
         }
+        
+        if let points = try? container.nestedContainer(keyedBy: RoutableLocationsKeys.self, forKey: .routableLocations),
+            let coordinatePairs = try points.decodeIfPresent([[CLLocationDegrees]].self, forKey: .points) {
+            let coordinates = coordinatePairs.map { CLLocationCoordinate2D(geoJSON: $0) }
```

Comment:
Adding support for Codable CLLocationCoordinate2D by adding an extension would be cleaner but wouldn't be worth it until the [decoding bottleneck](https://bugs.swift.org/browse/SR-6252) is fixed.